### PR TITLE
feat: start-preview and stop-preview CLI commands

### PR DESCRIPTION
**Pull request by rephus**
Partially closes: #3223

### Description:

Screenshot from 2022-09-15 11-27-37
Screenshot from 2022-09-15 11-32-25

### CODE REVIEW

**Comment by rephus** [comment]
We need to "release" this command before I can impelment the github action. I'll wrote the documentation then

File: packages/cli/src/index.ts [hunk 1]
```diff
@@ -259,12 +263,59 @@ program
...
+    .option('--no-version-check')
+    .option(
+        '-s, --select <models...>',
+        'specify models (accepts dbt selection syntax)',
+    )
+    .option(
+        '-m, --models <models...>',
+        'specify models (accepts dbt selection syntax)',
+    )
+    .option('--name <preview name>', 'name for the preview project [required]')
```
Comment (ZeRego) [inline packages/cli/src/index.ts:306]: We should mention the name should be unique.

File: packages/cli/src/handlers/preview.ts [hunk 2]
```diff
@@ -130,3 +153,59 @@ export const previewHandler = async (
...
+
+    const previewProject = await getPreviewProject(projectName);
+    if (previewProject) {
+        await lightdashApi({
+            method: 'DELETE',
+            url: `/api/v1/org/projects/${previewProject.projectUuid}`,
+            body: undefined,
+        });
+        console.error(
+            `Successfully deleted preview project name ${projectName}`,
```
Comment (ZeRego) [inline packages/cli/src/handlers/preview.ts:204]:
```suggestion
            `Successfully deleted preview project named ${projectName}`,
```

**Review by ZeRego — APPROVED** [review]
Code looks good. 
Just got some small suggestions.
Would be good if we also add events for these commands.

**Pull request merged**